vaas-app/src/vaas/manager/models.py

```diff
@@ -122,6 +122,9 @@ def mode_constructor(self):
         elif self.mode == 'fallback':
             return 'fallback()'
 
+    def get_clusters(self):
```

Comment:
get_directors

Reply:
nah. Is get_clusters because we took of clusters in director model :)
So get_clusters is correct.

Reply:
right